assignment/scripts/function-practice.js

```diff
@@ -49,41 +65,110 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
-function getLast( array ) {
-
+function getLast(array) {
+  if(array === []){
```

Comment:
Since arrays are objects, and objects are _never_ `===` unless they are literally the same object in memory, this if block will never run.

Thankfully, you don't need it. Line 72 already returns `undefined` if the array is empty.